person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(name = 'Unknown', age, parent_permission = true)
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md) for this project, " `Create class Person with the following: Instance vars: @id, @name, and @age. ` ", please kindly add an @id instance variable to enable use to uniquely identify a person, some like so: 
```
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```
Please kindly that and please do not forget to add a getter method for it.